geoapi/tasks/streetview.py

```diff
@@ -0,0 +1,332 @@
+                  logItem: dict=None,
+                  remove_dir: bool=False):
+    StreetviewService.deleteInstance(streetview_instance.id)
+    progress_error(user, task_uuid, status, message, logItem)
+    if remove_dir:
+        # TODO: Change to include service after user id
+        remove_project_streetview_dir(user.id, task_uuid)
+
+def _from_tapis(user: User, task_uuid: UUID, systemId: str, path: str, organization_key: str):
+    client = AgaveUtils(user.jwt)
+    listing = client.listing(systemId, path)
+    files_in_directory = listing[1:]
+
+    base_filepath = get_project_streetview_dir(user.id, task_uuid)
+
+    # TODO Should handle retry
+    if not os.path.isdir(base_filepath):
+        make_project_streetview_dir(user.id, task_uuid)
+    else:
+        remove_project_streetview_dir(user.id, task_uuid)
+        make_project_streetview_dir(user.id, task_uuid)
+        NotificationsService.create(user,
+                                    "success",
+                                    "Cleaning up previous session before upload.")
+
+    img_list = []
+
+    done_files = 0
+    files_length = len(files_in_directory)
+
+    for item in files_in_directory:
+        if item.type == "dir":
+            NotificationsService.create(user,
+                                        "warning",
+                                        "Invalid upload type. Disregarding...")
+            continue
+        if item.path.suffix.lower().lstrip('.') not in features.FeaturesService.IMAGE_FILE_EXTENSIONS:
+            continue
+        try:
+            img_name = os.path.join(str(base_filepath), Path(item.path).name)
+            img_list.append(img_name)
+            client.getRawFileToPath(systemId, item.path, img_name)
+
+            done_files += 1
+
+            NotificationsService.updateProgress(task_uuid=task_uuid,
+                                                status="in_progress",
+                                                message="Transferring files from DesignSafe to geoapi",
+                                                progress=int(done_files / files_length * 100),
+                                                logItem={"uploadFiles": img_list})
+
+        except Exception as e:
+            done_files -= 1
+            raise Exception("Could not import file from agave: {} :: {}, {}" \
+                            .format(systemId, path, e))
+
+    if len(img_list) == 0:
+        raise Exception("No images have been uploaded to geoapi!")
+
+
+def _to_mapillary(user: User, streetview_instance: StreetviewInstance, task_uuid: UUID, organization_key: str):
+    streetview_service = streetview_instance.streetview
+    token = streetview_service.token
+
+    service_user = streetview_service.service_user
+
+    try:
+        NotificationsService.updateProgress(task_uuid,
+                                            "created",
+                                            "Started upload to Mapillary")
```

Comment:
```suggestion
                                            "Uploading to Mapillary")
```

suggestion is to have the same form as the previous notification